### ISSUE TITLE
User scripts not being run in 1.70 release

### ISSUE THREAD

**Issue by aharmat** [foxglove/studio#6823]
User scripts are not being executed in the 1.70 release of Studio. Output topic names don't appear in the relevant panels, and `log` statements put into the code don't print anything.

- Version: 1.70
- Platform: MacOS
- Data source: mcap, foxglove data storage

**Steps To Reproduce**
Try to run any user script in 1.70.
UPDATE: It seems that any user script must import something from `@foxglove/schemas` for this problem to appear.

**Expected Behavior**
User scripts execute as with previous releases.

**Comment by jtbandes** [comment]
I'm not able to reproduce this issue. Could you share more detailed steps to reproduce, such as sample data and a layout?

<img width="1195" alt="image" src="https://github.com/foxglove/studio/assets/14237/973ca299-d756-4c53-827f-fdaa69f2fa9f">

**Comment by aharmat** [comment]
I have experimented with the following layouts:
- Creating a new one
  - While signed in to my org
  - While signed out
- Modifying one from my organization
  - Saving a local copy
  - Not saving a local copy

An in all cases, what I have reduced it to is that the inclusion of any type from `@foxglove/schemas` in any user script stops all user scripts from running. For example, adding `import { LocationFix } from "@foxglove/schemas";` to the hello world script stops it from running. Or for another example, create two scripts, one the hello world sample, and another that only contains `import { Color } from "@foxglove/schemas";`. The hello world sample runs if you comment out the one import line in the 2nd script, but not if you have it uncommented.
